nupic/research/frameworks/vernon/experiments/cl_experiment.py

```diff
@@ -141,6 +156,162 @@ def compute_task_indices(cls, config, dataset):
                 task_indices[i].extend(class_indices[j + (i * num_classes_per_task)])
         return task_indices
 
+    def get_active_classes(self, task_num):
+        """
+        Returns a list of label indices that are "active" during training under the
+        continual learning scenario specified by the config. In the "task" scenario,
+        only the classes that are being trained are active. In the "class" scenario,
+        all tasks that the model has previously observed are active. More information
+        about active classes can be found here: https://arxiv.org/abs/1904.07734
+
+        :param task_num: zero-based index of the task
+        """
+        if self.cl_experiment_type == "task":
+            return [label for label in range(
+                self.num_classes_per_task * task_num,
+                self.num_classes_per_task * (task_num + 1)
+            )]
+        elif self.cl_experiment_type == "class":
+            return [label for label in range(
+                self.num_classes_per_task * (task_num + 1)
+            )]
+        elif self.cl_experiment_type == "domain":
+            raise NotImplementedError
```

Comment:
Maybe add a `#TODO` comment here so it's easy to search the file and see what tasks are left for later.

Reply:
done.

Reply:
Also repeated comment below. We have no plans to deal with the domain scenario, I don't see a reason to leave this TODO here.